### PR TITLE
Add DK parser and validator

### PR DESCRIPTION
**Pull request by ivorbosloper**
Implement https://github.com/fiboa/data/issues/15

### CODE REVIEW

File: tests/test_convert.py [hunk 1]
```diff
@@ -14,7 +14,7 @@ def out_file():
         yield out
 
 
-@mark.parametrize('converter', ['at', 'be_vlg', 'br_ba_lem', 'de_sh', 'ec_lv', 'ec_si', 'fi', 'fr', 'nl', 'nl_crop', 'pt'])
```
Comment (m-mohr) [inline tests/test_convert.py:17]: Is the removal of br_ba_lem intentional?

File: fiboa_cli/datasets/dk.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
+from ..convert_utils import convert as convert_
+
+SOURCES = "https://landbrugsgeodata.fvm.dk/Download/Marker/Marker_2023.zip"
+LAYER_FILTER = None
+ID = "dk"
+SHORT_NAME = "DK Marker"
+TITLE = "Denmark Crop Fields (Marker)"
+DESCRIPTION = """The Danish Ministry of Food, Agriculture and Fisheries publishes
+Crop Fields (Marker) for each year."""
```
Comment (m-mohr) [inline fiboa_cli/datasets/dk.py:9]:
```suggestion
DESCRIPTION = "The Danish Ministry of Food, Agriculture and Fisheries publishes Crop Fields (Marker) for each year."
```

File: fiboa_cli/datasets/dk.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
+from ..convert_utils import convert as convert_
+
+SOURCES = "https://landbrugsgeodata.fvm.dk/Download/Marker/Marker_2023.zip"
+LAYER_FILTER = None
+ID = "dk"
+SHORT_NAME = "DK Marker"
```
Comment (m-mohr) [inline fiboa_cli/datasets/dk.py:6]:
```suggestion
SHORT_NAME = "Denmark"
```

File: tests/test_convert.py (hunk 1 above)
Comment (ivorbosloper) [reply tests/test_convert.py:17]: Thanks for spotting this, a rebase/merge mistake

**Review by m-mohr — APPROVED** [review]
Thank you!

**Pull request merged**